src/main/java/io/relution/jenkins/awssqs/interfaces/SQSQueueListener.java

```diff
@@ -34,6 +34,11 @@
      */
     String getQueueUuid();
 
+    /**
+     * The disable flag of Amazon CodeCommit Check for listner
+     * @return The disable flag of Amazon CodeCommit Check for listner
+     */
+    boolean isDisableCodeCommit();
```

Comment:
breaks binary compatibility if the interface is implemented outside the plugin

Reply:
Default Method may help

Reply:
I used Default method.